react-app/.env.development

```diff
@@ -1,4 +1,5 @@
 REACT_APP_PAGE_MODEL_PATH=http://localhost:4502/content/we-retail-journal/react.model.json
 API_HOST=
-APP_ROOT_PATH: "/content/we-retail-journal/react";
+APP_ROOT_PATH: "/content/we-retail-journal/react"
+PARAMETER_LIMIT=5000000;
```

Comment:
Minor suggestion - Parameter limit can be reduced to a lower number (eg: 500000/100000)

Reply:
It depends on number of pages we have, I had tested with approximately 200 pages and this configuration works well.